Intro to Github Assignment

tam670 opened the pull request:
Used Drake's suggestions

bjones1 commented on Scrabble/tam670.py:
```diff
@@ -1,66 +1,141 @@
+# <span style="text-decoration: underline;">of</span> the valid words that I can
+# make with <span style="text-decoration: underline;">the</span> letters.
+#
+# <span style="background-color: #fbeeb8;">dsr209:</span> I would say something
```
Remove your partner's comments; instead, integrate these into your revision.

bjones1 commented on Scrabble/tam670.py:
```diff
@@ -1,66 +1,141 @@
+# calculated by tallying up the point values of each letter in the word (see
+# corresponding chart below).  My approach is to compare each word in the given
+# dictionary to my tiles, letter by letter. This will ensure that I can find all
+# <span style="text-decoration: underline;">of</span> the valid words that I can
```
Underline is messy in Markdown. Choose something else (_this_, for example, which is `_this_` in Markdown.).

tam670 commented:
Thanks for the feedback. I have removed all of Drake's comments, as well as removed all underlined text @bjones1

bjones1 reviewed (changes requested):
Move this to the `Scrabble/tam670` directory, then I'll approve.

tam670 commented:
line 12

**dsr209** I would say something at the end about how each letter correlates to a specific score then the score is totaled by adding each letters individual score together. I also added a few words to help with grammar and flow, I underlined them. After looking at the game rules on codingame, you need to include a better discription of the dictionary and how the user

**tam670** I added parts about the dictionary and how it is given. I also provided more context about how the score it calculated.

tam670 commented:
line 54 

**dsr209**: I would add more of a description paragraph under the inputs heading. Describe in detail what each input is and what it is for. I also think for the literate part of the programming, leaving the code separate from the comments is ideal. It allows the reader to read what is going to happen and why and then see the code that does it.

**tam670**: I have moved all the comments to above this set of code. I agree it does make the program more legible.

tam670 commented:
line 75


**dsr209**: I think the variables should be initialized at the beginning of the code, but the naming is descriptive—well done.

**tam670**: I have decided to make no changes here, since this is the beginning of the code.

tam670 commented:
line 52 

**dsr209**: How does this loop work? Where is "word" initialized? Is it just a part of the game?

**tam670**: I am adding context to this below.

tam670 commented:
line 120 


**dsr209**: Overall, I think the writing is mostly clear and coherent but could use a bit of work. There is only one paragraph, and that is the problem and approach paragraph. This paragraph conveys most of the information needed. There are several grammatical errors and at least one misspelling. The sentences were missing words or not worded very well. The code in CodeChat is difficult to read with the comments and the headers left aligned. The code itself is good and worked when I ran it on Codingame.

**tam670**: I appreciated the suggestions, Drake. I really like the idea of having the comments at the beginning of a code block, with the same indentation as the code. I think that you would be very satisfied with the changes that I implemented.

tam670 commented:
@bjones1 I have revised my code, and moved all of @DrakeRobertson 's comments to this ticket. I am requesting another review

bjones1 commented:
Looks good!
